stdlib/asm/crypto/stark/ood_frames.masm

```diff
@@ -16,100 +16,104 @@ use.std::crypto::stark::random_coin
 #!
 #! 1. n, m and l are the widths of the main segment, auxiliary segment and constraint composition
 #!    traces, respectively.
-#! 2. S_i are the evaluations of columns in the main segment, auxiliary segment and constraint composition
-#!    at the the out-of-domain point z.
-#! 3. T_i are the evaluations of columns in the main segment, auxiliary segment and constraint composition
-#!    at the the out-of-domain point gz.
+#! 2. S_i are the evaluations of columns in the main segment, auxiliary segment and constraint
+#!    composition at the the out-of-domain point z.
+#! 3. T_i are the evaluations of columns in the main segment, auxiliary segment and constraint
+#!    composition at the the out-of-domain point gz.
 #! 4. alpha is the randomness used in order to build the DEEP polynomial.
+#!
+#! Inputs:  []
+#! Outputs: []
```

Comment:
It's not really clear "where" this computation happens. Could we mention where in memory we store the results?

Reply:
Added a comment about where and how it is saved in memory